features/user/login.feature

```diff
@@ -3,6 +3,7 @@ Feature:
     As a user
     I must be able to log in
 
+    @javascript
```

Comment:
This scenario fails on my computer, event after a fresh `make`
```
  @javascript
  Scenario: As anonymous, I cannot access to the homepage # features/user/login.feature:7
    When I go to the homepage                             # Behat\MinkExtension\Context\MinkContext::iAmOnHomepage()
      invalid argument
        (Session info: headless chrome=81.0.4044.113) (Facebook\WebDriver\Exception\InvalidArgumentException)
```

Reply:
I've pushed a fix, could you try it locally?